webrtc-stats.html

```diff
@@ -2580,6 +2648,14 @@ <h2>
                   <code>Math.sqrt(0.0026/0.02) = 0.36</code>, which is the same value that would be
                   obtained by doing an RMS calculation over the contiguous 20ms segment of audio.
                 </p>
+                <p>
+                  Note: If multiple audio channels are used, metrics based on
+                  samples do not increment at a higher rate. "A sample" refers
+                  to having a sample in either channel - simultaneously having
+                  samples in multiple channels counts as a single sample. The
+                  "audio energy of a sample" refers to the highest energy of any
+                  channel.
```

Comment:
confirming if "highest audio energy" was intended, we should explain what is the consequence or why we picked the highest. I am assuming this is not documented in any other standards.